front/src/main.js

```diff
@@ -48,131 +48,170 @@ const layout = {
     }
 };
 
-
 Plotly.newPlot('plotlyDiv', [], layout);
 
-const dataName = 'data.json';
-// const dataName = 'convection-diffusion.json';
-const scalarFetch = fetch(dataName).then(response => response.json());
-const gradientFetch = fetch('gradient-fields.json').then(response => response.json());
-let intervalId = 0;
+let renderer = {
+    type : "cycle", //0-cycle, 1-slider
+    dataType : 0, //0-diffusion 1-convaction 2-convaction-diffusion
+    dataName : [ "diffusion", "convaction", "convaction-diffusion"],
+    typeFile : [
+        ["data.json", "gradient-fields.json"],
```

Comment:
data.json is not a good file name, so please change "data.json" to "diffusion.json" by updating 'Python code'

Reply:
finished